doc/index.rst

```diff
@@ -1,6 +1,6 @@
-####################
-Spyder Documentation
-####################
+###############################
+Welcome to Spyder Documentation
```

Comment:
What if change this to `Welcome to our Documentation` to make the title a bit friendlier?

I mean, if people landed on this page, they know for sure they are reading the Spyder docs, right?

Reply:
Also, (if you don't like my previous suggestion) `Welcome to Spyder Documentation` sounds a bit odd to me.

Could we change it to `Welcome to the Spyder Documentation`? Is there something wrong with that?